option.go

```diff
@@ -95,6 +96,14 @@ func WithSQLCommenter(enabled bool) Option {
 	})
 }
 
+// WithTextMapPropagator specifies a text map propagator to used by the SQLCommentor
+// option. If none is specified, the global text map propagator is used.
```

Comment:
```suggestion
// WithTextMapPropagator specifies a text map propagator to used by the SQLCommenter
// option. If none is specified, the global text map propagator is used.
//
// Notice: This option is EXPERIMENTAL and may be changed or removed in a
// later release.
```

Since it is an option tied to the SQL SQLCommenter, it should be marked as EXPERIMENTAL.